src/utils.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export const newsApi = axios.create({
+    baseURL: `https://jack-nc-news-ap.onrender.com/api/`
+})
+
+export const fetchArticles = () => {
+    return newsApi
+    .get('articles')
+    .then(({data}) => {
+        return data;
+    })
+    .catch((err) => {
+        console.log(err);
+    })
```

Comment:
I'd recommend taking this catch out of here. If you want to catch the error do it at the end of the promise chain. (In the component) We'll talk more about error handling later in the week 👍